dvc/exceptions.py

```diff
@@ -303,3 +303,21 @@ def __init__(self, hook_name):
             "https://man.dvc.org/install "
             "for more info.".format(hook_name)
         )
+
+
+class FailedDownloadError(DvcException):
```

Comment:
`Failed` and `Error` makes it double negative - successful download. How about
```suggestion
class FetchError(DvcException):
```
or
```suggestion
class PullError(DvcException):
```
to better reflect what is going on. Or maybe there is something even better.

Reply:
Hm, or maybe
```suggestion
class DownloadError(DvcException):
```
indeed.

Reply:
:+1: